rorapi/v2/models.py

```diff
@@ -130,4 +133,40 @@ class MatchingResult:
 
     def __init__(self, data):
         self.number_of_results = len(data)
-        self.items = [MatchedOrganization(x) for x in data]
\ No newline at end of file
+        self.items = [MatchedOrganization(x) for x in data]
+
+
+class Client(models.Model):
+    # Required fields
+    email = models.EmailField(max_length=255, unique=True)
```

Comment:
Per the spec, email does NOT need to be unique - users should be able to register multiple client IDs with the same email.